src/eterna/folding/__tests__/NuPACKDev.test.ts

```diff
@@ -0,0 +1,232 @@
+//import Folder, {MultiFoldResult} from '../Folder';
+import Folder, {DefectResult, SuboptEnsembleResult} from '../Folder';
+import NuPACK from '../NuPACK';
+//import Vienna from '../Vienna';
+//import Vienna2 from '../Vienna2';
+//import LinearFoldV from '../LinearFoldV';
+import './jest-matcher-deep-close-to';
+//import SecStruct from 'eterna/rnatypes/SecStruct';
+import Sequence from 'eterna/rnatypes/Sequence';
+import SecStruct from 'eterna/rnatypes/SecStruct';
+//import {Oligo} from 'eterna/rnatypes/Oligo';
+
+function CreateFolder(type: any): Promise<Folder | null> {
+    return type.create();
+}
+
+/*
+
+test('NuPACK:suboptstructuresNoOligos', () => {        
```

Comment:
Are we able to remove all the commented out tests? Otherwise, should they be (modified and) enabled?

Reply:
I left them there for future troubleshooting help but I can pull those out np as they are covered in folder.test.ts i think

Reply:
i took them out